applications/esm/Dockerfile.base

```diff
@@ -34,15 +33,26 @@ USER $SERVICE_NAME
 RUN wget --no-check-certificate "https://github.com/conda-forge/miniforge/releases/download/24.3.0-0/Miniforge3-$(uname)-$(uname -m).sh"
 RUN bash Miniforge3-$(uname)-$(uname -m).sh -b -p "${HOME}/conda"
 
+WORKDIR /app
+
 RUN git clone --recursive https://github.com/facebookresearch/esm.git
-COPY esm_change_all.patch /app/esm/esm_change_all.patch
 WORKDIR /app/esm
-RUN git apply esm_change_all.patch
+RUN git checkout -b esm 2b369911bb5b4b0dda914521b9475cad1656b2ac
+
+
+RUN wget https://github.com/IntelLabs/Open-Omics-Acceleration-Framework/releases/download/3.0/Source_code_with_submodules.tar.gz 
+RUN tar -xzf Source_code_with_submodules.tar.gz
+WORKDIR /app/esm
```

Comment:
Since we are already in /app/esm, we do not need to do it again,

Reply:
Removed WORKDIR from Dockerfile.base in ESM